src/test/java/com/synopsys/integration/jenkins/wrapper/JenkinsProxyHelperTest.java

```diff
@@ -24,17 +24,17 @@
 import jenkins.model.Jenkins;
 
 public class JenkinsProxyHelperTest {
-    final static String ntlmDomain = "ntlmDomain";
-
-    final String proxyHost = "proxyHost";
-    final String proxyUsername = "proxyUsername";
-    final String proxyPassword = "proxyPassword";
-    final String ntlmWorkstation = "ntlmWorkstation";
-    final int proxyPort = 13;
+    final static String expectedNtlmDomain = "ntlmDomain";
```

Comment:
Most of the changes are due to naming changes. I will try to add a comment to all those that are an actual logic change.